app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PasswordResetsController < ApplicationController
+  require_unauthenticated_access
+  before_action :ensure_manual_login_allowed
+
+  layout "public"
+
+  def new
+  end
+
+  def create
+    os = request.operating_system
+    browser = request.browser
+    SendResetPasswordEmailJob.perform_later(params[:email], os, browser) # queue as a job to avoid timing attacks
+
+    redirect_to login_path, notice: "If an account with that email was found, we have sent a link to reset the password"
```

Comment:
copyediting to be a little shorter for quick read and adding period (I really need to make these toast messages be persistent and not slide off so quickly):

```suggestion
    redirect_to login_path, notice: "If that email is valid, we just sent password reset email."
```